vsphere/vmdk-to-ovf.sh

```diff
@@ -7,8 +7,8 @@ NOCOLOR='\033[0m'
 [ "$DEBUG" == 'true' ] && set -x
 
 ARGC=$#
-if [ $ARGC -ne 7 ]; then
-    echo "USAGE: $(basename $0) <vCenter Server> <vCenter Username> <vCenter Password> <vCenter Cluster> <vCenter Datastore> <Template Name> <Path to VMDK>"
+if [ $ARGC -lt 7 ]; then
```

Comment:
If we miss a parameter but provide the optional parameter, we end up with VMDK_PATH set to "--disableefi"
Obviously, this should cause a visible error pretty quick, but maybe we can check for it right away and prevent issues?

```suggestion
if [ $ARGC -lt 7 || "$7" = "--disableefi"]; then
```